### PR TITLE
Unit tests update

### PR DESCRIPTION
**Pull request by tumf87**
CRUD Panel update trait unit tests.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ All Notable changes to `Backpack CRUD` will be documented in this file
 - Nothing
 
 
+## [3.2.20] - 2017-09-x
+
+## Fixed
+- zh-cn instead of zh-CN language folder - fixes #849;
```
Comment (lloy0076) [inline CHANGELOG.md]: https://github.com/Laravel-Backpack/CRUD/issues/849 is fixed on dev; apparently.

**Comment by tumf87** [comment]
Wait, @tabacitu, why is this a pull request directly to the master branch? Did you change it or did I missed it? :)

File: CHANGELOG.md (hunk 1 above)
Comment (tabacitu) [reply CHANGELOG.md]: Merging it there now.

**Comment by tabacitu** [comment]
Thanks a lot for the unit tests, @tumf87 , and the reviews, @lloy0076 . I just merged all the unit tests into the ```dev``` branch since they're all non-breaking-changes. This should allow:
- @tumf87 to work on the bugs he's discovered while writing the unit tests
- @lloy0076 to comment about unit tests into one spot (once we do the "Dev into master" PR)
- me to see the diff correctly (for some reason github showed a huge diff even though I merged the first PR into dev)

Thanks a lot of collaborating on this, you guys. I think this is a BIG step up for Backpack. Stability seems within reach now :-)

**Pull request merged**